src/ontology/lexmatch/unmapped_ncit_lex.tsv

```diff
@@ -1,115 +1,60 @@
 subject_id	subject_label	predicate_id	object_id	object_label	mapping_justification	mapping_tool	confidence	subject_match_field	object_match_field	match_string	comment
-ID		A oboInOwl:hasDbXref	>A oboInOwl:source	>A sssom:object_label							
+ID		>A oboInOwl:source	A oboInOwl:hasDbXref								
```

Comment:
Wrong column order - needs to be ....`object_id predicate_id`.. because of the `>A` operator in the ROBOT template.